test/forms/DateFormDataSpec.scala

```diff
@@ -38,6 +38,10 @@ class DateFormDataSpec extends UnitTest {
       aDateFormData.copy(day = "1", month = "1", year = "2022").isValidLocalDate shouldBe true
     }
 
+    "return true when data correct even with spaces" in {
+      aDateFormData.copy(day = "1", month = "1", year = "20 22").isValidLocalDate shouldBe true
```

Comment:
Let's pick double digits for every field, like:
`day = "1 0"`, month = "1 2"` 
so we are sure we test for all cases

Reply:
:+1: